lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -513,7 +513,7 @@ class ServiceFabrikAdminController extends BaseController {
       throw new BadRequest('repeatInterval | type are mandatory');
     }
     const boshDirectorName = req.body.bosh_director;
-    const boshDirector = bosh.getBoshDirectorByName(boshDirectorName);
+    const boshDirector = new bosh.BoshDirectorClient();
```

Comment:
wrong. Should refer to the same instance which is created in index.js. Just refer like this
```
bosh. boshDirectorClient;
```